src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -85,9 +85,14 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
         {
             var elements = new List<NodeSearchElement>();
             var inputPortType = PortViewModel.PortModel.GetInputPortType();
+
+            //List of nodes that are skipped temporarily, and will display list of default suggestions instead.
+            var skippedInputTypes = new List<string>() { "var", "object", "string", "bool", "int", "double" };
+          
+            if (inputPortType == null || (skippedInputTypes.Any(s => s == inputPortType.ToString()))) return elements;
```

Comment:
I think @mjkkirschner also suggested using {} and follow our own coding style documented.